mmv1/products/apigee/EnvironmentAddonsConfig.yaml

```
@@ -0,0 +1,74 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'EnvironmentAddonsConfig'
+api_resource_type_kind: AddonsConfig
+description: Enable/Disable add-ons for an Apigee environment.
+references:
+  guides:
+    'Creating an environment': 'https://cloud.google.com/apigee/docs/api-platform/get-started/create-environment'
+  api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.addonsConfig/setAddonEnablement'
+docs:
+base_url: '{{env_id}}/addonsConfig'
+self_link: '{{env_id}}/addonsConfig'
+update_url: '{{env_id}}/addonsConfig:setAddonEnablement'
+update_verb: 'POST'
+timeouts:
+  update_minutes: 5
+async:
+  actions: ['update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+    path: 'name'
+    wait_ms: 1000
+  result:
+    path: 'response'
+    resource_inside_response: false
+  error:
+    path: 'error'
+    message: 'message'
```

Most of these are unused - see https://github.com/hashicorp/terraform-provider-google/issues/17098 for more details.

```suggestion
async:
  actions: ['update']
  type: 'OpAsync'
  operation:
    base_url: '{{op_id}}'
```